src/makeRobotAccountant.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
-}
+  let count = 0;
+
+  const result = (a) => {
+    count++;
+
+    const sum = (b) => {
```

Comment:
You can add 0 as a default value to a and b parameters . In case there  is no argument is passed, it would still work.